ragna/source_storages/_chroma.py

```diff
@@ -127,11 +125,11 @@ def retrieve(
         return self._take_sources_up_to_max_tokens(
             (
                 Source(
-                    id=result["id"],
-                    document=document_map[result["metadata"]["document_id"]],
-                    location=result["metadata"]["page_numbers"],
-                    content=result["document"],
-                    num_tokens=result["metadata"]["num_tokens"],
+                    id=result["ids"],
```

Comment:
Question: Is the addition of an 's' to all keys a `chroma` thing or a `ragna` thing? And is it correct to add the `s` to the id field as well?

Reply:
See https://github.com/Quansight/ragna/pull/435#discussion_r1654976899.